samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties

```diff
@@ -385,6 +385,8 @@ seb_exam_keys_upload_info = Fitxategi bidez igotako konfigurazioetan, azterketar
 Deskargatu egin beharko duzu berriro konfigurazioko fitxategia (aldatuta) eta txertatu beharko dituzu horretan dauden gakoak.
 seb_exam_keys_upload_published_info = Sartu azterketaren gakoak, konfigurazioko fitxategian daudenak. Fitxategia goiko aldean deskargatu daiteke. \
 Konfigurazioa aldatu da azterketa argitaratu denean.
+seb_exam_keys_recommended=It is recommended to add exam keys for this type of configuration, because the Safe Exam Browser Configuration can \
+not be validated automatically
```

Comment:
Hi @PabloSanRoman, could you help me with this translation please?

Reply:
Hi @stetsche , sure!. 

seb_exam_keys_recommended=Azterketara sartzeko pasahitza jartzea gomendagarria da, Safe Exam Brower-en konfigurazioa agian ez baita automatikoki balidatzen.

Reply:
Thank you!